### ISSUE TITLE
Autoprefixer issues warning on fill-available in SCSS file (v4.0.1)

### ISSUE THREAD

**Issue by Whyves** [DominicTobias/react-image-crop#176]
Hi,

I'm compiling the scss file using webpack and the autoprefixer issues a warning on line 40:

 `max-height: -webkit-fill-available;`

I noticed that you did change the fill-available to stretch on version 4.0.1 but I guess that autoprefixer doesn't care that it's already prefixed properly. 

Since you're also using webpack to compile the project, any reason why you're not letting it autoprefix the final css? If we remove the prefixed line in the scss file, autoprefixer does its job and doesn't complain anymore.

Thanks!

**Comment by bogdansoare** [comment]
also having this issue
```
(Emitted value instead of an instance of Error) autoprefixer: /Users/bogdansoare/Sites/platform_workflow/node_modules/react-image-crop/dist/ReactCrop.css:15:5: Replace fill-available to stretch, because spec had been changed
```

**Comment by DominicTobias** [comment]
Hi the project at the moment is using `node-sass` and the css isn't loading via webpack (through the JS file). I will keep in mind to see if I can get auto prefixing working with `node-sass` for a future release.

However that won't help @bogdansoare issue at least, since the error is coming from the css file, and I'm not really sure why as I followed the output that autoprefixer supposedly generates:

https://github.com/postcss/autoprefixer/issues/1035#issuecomment-387247093

```css
max-height: -webkit-fill-available;
max-height: -moz-available;
max-height: stretch;
```

**Comment by Whyves** [comment]
@DominicTobias , Thanks for considering it. In the mean time, I copied the scss file, removed the offending lines and prefix it via webpack so this solve my immediate needs. Just it would be nice to be able to use the files that are part of the package :-)

**Comment by csuich2** [comment]
I hit this same issue but worked around it by putting

```css
/* autoprefixer: off */
```

before my @import. *Maybe* this is causing problems I don't notice, but things seem to work fine.

**Comment by DominicTobias** [comment]
If one of you doesn't mind please try putting `/* autoprefixer: off */` in the scss file here:

```css
&__image {
    /* autoprefixer: off */
    display: block;
    max-width: 100%;
    max-height: -webkit-fill-available;
    max-height: -moz-available;
    max-height: stretch;
  }
```

**Comment by DominicTobias** [comment]
Released in 4.0.2

**Comment by josh231101** [comment]
This has not been updated...
